source/boundary_temperature/function.cc

```diff
@@ -85,6 +108,24 @@ namespace aspect
       {
         prm.enter_subsection("Function");
         {
+          /**
+           * Choose the coordinates to evaluate the maximum refinement level
+           * function. The function can be declared in dependence of depth,
+           * cartesian coordinates or spherical coordinates. Note that the order
+           * of spherical coordinates is r,phi,theta and not r,theta,phi, since
+           * this allows for dimension independent expressions.
+           */
+          prm.declare_entry ("Coordinate system", "cartesian",
+                             Patterns::Selection ("cartesian|spherical|depth"),
+                             "A selection that determines the assumed coordinate "
+                             "system for the function variables. Allowed values "
+                             "are 'cartesian', 'spherical', and 'depth'. 'spherical' coordinates "
+                             "are interpreted as r,phi or r,phi,theta in 2D/3D "
+                             "respectively with theta being the polar angle. 'depth' "
+                             "will create a function, in which only the first "
+                             "variable is non-zero, which is interpreted to "
```

Comment:
I would change this to: "... in which only the first parameter is non-zero ...". "Variable" suggests the function only *outputs* one component.